modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/experience/components/ExperienceToolbarSection.test.js

```diff
@@ -98,6 +98,7 @@ const mockState = {
 		}
 	},
 	permissions: {
+		EDIT_SEGMENTS_ENTRY: true,
```

Comment:
cc @andresfulla

I see you last touched this area in c63903b3de636e08289dc9b8f14d2874be56cbf7 but it looks like we needed to do the same for `EDIT_SEGMENTS_ENTRY` as well.

Reply:
thanks!